app/controllers/planning_application/permitted_development_rights_controller.rb

```diff
@@ -73,6 +73,9 @@ def set_planning_application
       planning_application = planning_applications_scope.find(planning_application_id)
 
       @planning_application = PlanningApplicationPresenter.new(view_context, planning_application)
+      @evidence_groups = planning_application&.immunity_detail&.evidence_groups&.map do |group|
+        EvidenceGroupPresenter.new(view_context, group)
+      end
```

Comment:
Wondering if this might be nicer in it's own method, kind of like `set_review_immunity_detail_and_form` that's in this file. Seems a bit cleaner and if you return if it's not immune then we don't need all the "&"s